Lesson2/HT2_1-stack/HT2_1-stack/HT2_1main.cpp

```diff
@@ -0,0 +1,100 @@
+#include <iostream>
+#include <string>
+#include <stack>
+
+#include "HT2_1head.hpp"
+
+void EnterString (std::string &MyString){
+    getline(std::cin, MyString);
+};
+
+bool CheckValidString (std::string const & MyString, std::stack<char> & MyStack){
```

Comment:
**you didn't understood the task again - you should read the chat about the signature**